src/encoded/viewconfigs/matrix.py

```diff
@@ -44,10 +44,7 @@ def __init__(self, context, request):
     @staticmethod
     def _set_result_title(type_info):
         '''Helper function for class and child classes'''
-        if type_info.name == 'Annotation':
-            title = 'Encyclopedia'
-        else:
-            title = type_info.name + ' Matrix'
+        title = type_info.name + ' matrix'
```

Comment:
This is a method children classes can use. It may be  good idea to sanitize (like check for falsey) and check for None (null)...

Reply:
This may be a question for Jason/Idan. ENCD-4244 enables users to view page: audit/type=Annotation. Should that page have "matrix" in its title (see auditview.py)